runtime/common/src/integration_tests.rs

```diff
@@ -953,43 +1138,44 @@ fn crowdloan_ending_period_bid() {
 		run_to_block(100);
 
 		assert_eq!(Auctions::auction_status(100), AuctionStatus::<u32>::EndingPeriod(0, 0));
-		let mut winning = [None; SlotRange::SLOT_RANGE_COUNT];
-		winning[SlotRange::ZeroOne as u8 as usize] = Some((2, ParaId::from(2001), 900));
+		let mut winning = [(); SlotRange::SLOT_RANGE_COUNT].map(|_| None);
```

Comment:
dq: what is `.map(|_| None)` doing here?

Reply:
this is some really clever shit I found online. Basically, it is really hard to define an array here without specifying the full type for an `Option`, even if all the values of the option are `None`. This tricks the compiler to allowing it.

Reply:
https://stackoverflow.com/a/69756635/1396977